### PR TITLE
basic contact us page

### LINKED ISSUE #30: Create Contact Us page
- [x] Thank you page

### PR DESCRIPTION
**Pull request by ajstyle2410**
This is  Contact  us page using bootstrap
 closes #30

### CODE REVIEW

File: css/contact_us.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+section {
```
Comment (ajayg2808) [inline css/contact_us.css:1]: No need to add **css** for contact us

File: contact_us.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                                <a href="#"> <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" width="30"
+                                        height="30">
+                                        <path
+                                            d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z" />
```
Comment (ajayg2808) [inline contact_us.html:52]: Replace SVG's with bootstrap icons

File: contact_us.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
+<!doctype html>
+<html>
+
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Arc-i-Tech </title>
```
Comment (ajayg2808) [inline contact_us.html:7]: Remove extra space from **title**

**Review by ajayg2808 — CHANGES_REQUESTED** [review]
- [ ] Use `main.css` and `main.js` this files are common for all pages.
- [ ] Use bootstrap icons
- [ ] Status bar not correct (Because `main.css` not included)
- [ ] `<head>` tag should be similar to `index.html`
- [ ] Check inline review (for files changed)

File: contact_us.html (hunk 2 above)
Comment (ajstyle2410) [reply contact_us.html:52]: i am replaced svg icons

File: contact_us.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
```
Comment (ajayg2808) [inline contact_us.html:7]: Remove extra space from title

File: index.html [hunk 5]
```diff
@@ -91,12 +91,12 @@ <h6 class="mx-auto p-2 rounded-3 theme-bg-dark-secondary w-fit-content">Status:
                   </div>
                 </div>
               </div>
-              <button class="carousel-control-prev border border-dark" type="button"
+              <button class="carousel-control-prev border border-dark z-2" type="button"
```
Comment (ajayg2808) [inline index.html:94]: Why `z-index` updated?

File: index.html [hunk 6]
```diff
@@ -91,12 +91,12 @@ <h6 class="mx-auto p-2 rounded-3 theme-bg-dark-secondary w-fit-content">Status:
...
                 data-bs-target="#project-profile-carousel" data-bs-slide="prev">
                 <span class="carousel-control-prev-icon" aria-hidden="true"></span>
                 <span class="visually-hidden">Previous</span>
               </button>
-              <button class="carousel-control-next border border-dark" type="button"
+              <button class="carousel-control-next border border-dark z-2" type="button"
```
Comment (ajayg2808) [inline index.html:99]: Why `z-index` updated?

**Review by ajayg2808 — CHANGES_REQUESTED** [review]
1. Contact us page looks good, but white strip around is not that good looking.
    image

2. Border in footer not working as expected in contact us page.
    image

3. Why _index.html_ committed here?

**Review by ajayg2808 — APPROVED** [review]

**Pull request merged**